tests/Http/Controllers/Api/Entries/Upload/Internal/UploadWebControllerBulkUploadsTest.php

```
@@ -0,0 +1,441 @@
+*/
+
+class UploadWebControllerBulkUploadsTest extends TestCase
+{
+    use DatabaseTransactions;
+    use Assertions;
+
+    private User $user;
+    private array $projectDefinition;
+    private Project $project;
+    private EntryGenerator $entryGenerator;
+    private string $endpoint = 'api/internal/bulk-upload/';
```

PHPMD (2.15.0)

Avoid unused private fields such as '$endpoint'. (Unused Code Rules)

(UnusedPrivateField)